torch_agents/cogment_verse_torch_agents/hive_adapter/hive_agent_adapter.py

```diff
@@ -16,15 +16,17 @@
 
 from cogment_verse import AgentAdapter
 
-from cogment_verse_torch_agents.atari_cnn import NatureAtariDQNModel
```

Comment:
Why did you remove that?

Reply:
It's not supposed to be agent/model. It is just a different qnet that can be passed as an argument.

Reply:
Ok, then did you migrate the configured params that were using that to this other approach?